apps/condo/domains/ticket/components/BaseTicketForm/TicketAssignments.tsx

```diff
@@ -144,11 +147,11 @@ const TicketAssignments = ({
 
     return (
         <Col span={24}>
-            <Row gutter={[0, 24]}>
+            <Row gutter={[0, 8]}>
```

Comment:
Move to constant like `const TICKET_ASSIGMENTS_ROW_GUTTER: [Gutter, Gutter] = [0, 8]`